cmd/util.go

```diff
@@ -0,0 +1,10 @@
+package cmd
+
+import "github.com/pkg/errors"
+
+func validateFiles(files []string) error {
```

Comment:
Function name validate is prematurely generic, I mean we can name it specific and make it generic if we are adding more checks later.